docs/index.md

```
@@ -14,6 +14,10 @@ Bringing ✨<b><i>interactivity</i></b>✨ to plotnine.
 
 </div>
 
+![](https://github.com/y-sunflower/ninejs/blob/main/coverage-badge.svg)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)